ql/src/test/org/apache/hadoop/hive/ql/testutil/ReflectionUtil.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hive.ql.testutil;
+
+import org.apache.commons.lang3.reflect.FieldUtils;
+import org.apache.hadoop.hive.ql.txn.compactor.Cleaner;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+
+import static java.lang.String.format;
+
+/**
+ * Various helper methods for mocking.
+ * <b>Please use it only in tests, not in production code!</b>
+ */
+public class ReflectionUtil {
```

Comment:
There is already a ReflectionUtil here - 
https://github.com/apache/hive/blob/master/common/src/java/org/apache/hive/common/util/ReflectionUtil.java

I think we must extend that rather than creating a new class.

Reply:
As I see, the only common thing in the ReflectionUtil class in the common module and this, is the name of the class. Anyway, it can be moved. But again, based on their code, they have way different purpose.

Reply:
I think its best to add in the existing file since other modules can also use it from `hive-common` rather than putting this in  `hive-exec`.

Reply:
Refactored.